### ISSUE TITLE
support MathJax v3 on the client

### ISSUE THREAD

**Issue by pkra** [pkra/mathjax-img#7]
Would be nice.

**Comment by pkra** [comment]
Time to do this so lurkers feel free to chime in.

I still like the idea of using esm (as in https://github.com/krautzource/mathjax-aria-label) but besides not having documented what I did there, the MathJax team said the hacky loading (https://github.com/krautzource/mathjax-aria-label/blob/master/docs/index.html#L32-L51) might cause issues. Not sure if it's salvageable. 

The "official" path is webpack etc http://docs.mathjax.org/en/latest/web/webpack.html#a-custom-extension.

**Comment by pkra** [comment]
Initial draft at https://github.com/pkra/mathjax-img/pull/12 (see also https://codepen.io/pkra/pen/qBRvjyK).

Using webpack seems like overkill :roll_eyes: I'll look if the hack in mathjax-aria-label can be salvaged.
